src/Models/Meetup.php

```diff
@@ -44,7 +45,7 @@ class Meetup extends Model
 
     public function getLogoPath(): string
```

Comment:
Reconsider using an accessor here. If you would name this method getLogoPathAttribute you could use it in your code as $model->logoPath later.